tf/main.tf

```diff
@@ -121,3 +121,15 @@ resource "cloudflare_access_policy" "policy" {
     email = ["CHANGEME@example.com"]
   }
 }
+
+resource "cloudflare_zone_settings_override" "wildebeest_zone_config" {
+    zone_id = var.cloudflare_zone_id
+    settings {
+        brotli = "on"
+        minify {
+            css = "on"
```

Comment:
why brotli/css is on? Can't we just switch JS off?

Reply:
I'd prefer to run with Brotli on and keep HTML unminified (better for the web; less issues). We should be enabling Brotli by default anyway.